locked-asset/simple-lock/src/proxy_farm.rs

```diff
@@ -153,8 +138,7 @@ pub trait ProxyFarmModule:
             ));
         }
 
-        let farm_address =
-            self.try_get_farm_address(&lp_proxy_token_attributes.lp_token_id, farm_type);
+        let farm_address = self.try_get_farm_address(&lp_proxy_token_attributes.lp_token_id);
         let enter_farm_result = self.call_farm_enter(
```

Comment:
If `payments` len is 1 (there are no additional farm payments, only the `proxy_lp_payment`), the `additional_farm_payments` will be empty and the SC will call endpoint `call_farm_enter`. It doesn't seem to affect anything, only gas spent redundantly.